test/e2e/testdata/wf-template-status-failed-conditional-metric.yaml

```
@@ -0,0 +1,31 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  generateName: wf-failed-conditional-metric-
```

```suggestion
  generateName: template-status-failed-conditional-metric-
```

again this should match the filename